File: dock_spawn/containers/vertical_dock_container.dart
part of dock_spawn;

class VerticalDockContainer extends SplitterDockContainer {

  VerticalDockContainer(DockManager dockManager, List<IDockContainer> childContainers) 
      : super(getNextId("vertical_splitter_"), dockManager, childContainers) {
    containerType = "vertical";
    
  }
  
  bool get stackedVertical {
    return true;
  }
}

File: js/lib/containers/VerticalDockContainer.js
import "SplitterDockContainer";

dockspawn.VerticalDockContainer = function(dockManager, childContainers)
{
    this.stackedVertical = true;
    dockspawn.SplitterDockContainer.call(this, getNextId("vertical_splitter_"), dockManager, childContainers);
    this.containerType = "vertical";
};
dockspawn.VerticalDockContainer.prototype = new dockspawn.SplitterDockContainer();
dockspawn.VerticalDockContainer.prototype.constructor = dockspawn.VerticalDockContainer;
